vcd/resource_vcd_lb_virtual_server_test.go

```diff
@@ -0,0 +1,258 @@
+// +build gateway lb lbVirtualServer ALL functional
+
+package vcd
+
+import (
+	"fmt"
+	"regexp"
+	"strings"
+	"testing"
+
+	"github.com/vmware/go-vcloud-director/v2/govcd"
+
+	"github.com/hashicorp/terraform/helper/resource"
+	"github.com/hashicorp/terraform/terraform"
+)
+
+func TestAccVcdLbVirtualServer(t *testing.T) {
+	// String map to fill the template
+	var params = StringMap{
+		"Org":               testConfig.VCD.Org,
+		"Vdc":               testConfig.VCD.Vdc,
+		"EdgeGateway":       testConfig.Networking.EdgeGateway,
+		"EdgeGatewayIp":     testConfig.Networking.ExternalIp,
+		"VirtualServerName": t.Name(),
+		"Tags":              "lb lbVirtualServer",
+	}
+
+	configText := templateFill(testAccVcdLbVirtualServer_step0, params)
+	debugPrintf("#[DEBUG] CONFIGURATION for step 0: %s", configText)
+
+	params["FuncName"] = t.Name() + "-step1"
+	configText2 := templateFill(testAccVcdLbVirtualServer_step2, params)
```

Comment:
where is step 1 ? :)

Reply:
Step 1 is the `import` test. I'm naming it so that step numbers are correct when error comes out